config/default/pv_claim.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Allegedly, persistent volumes are slow in some platforms; if this seriously affects the startup time, I would move the persistent volume config to an example (as an optional kustomization, say).

Reply:
We should use an EmptyDir volume here like we do for source-controller, PVC config could be made available in flux install/bootstrap.